packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -112,6 +112,16 @@
   display: flex;
   width: 100%;
   box-sizing: border-box;
+  &:before {
```

Comment:
For adding a ::before border? Hmm, no I am not seeing any use of a mixin that deals with ::befores

All of the ::befores are coded independently. We might be able to move all of them into a mixins, with a 10+ properties to configure, but that is probably out of scope for this small bug request.